src/plugins/vis_type_timeseries/server/lib/vis_data/request_processors/series/query.js

```diff
@@ -27,8 +27,9 @@ export function query(req, panel, series, esQueryConfig, indexPatternObject) {
     const { from, to } = offsetTime(req, series.offset_time);
 
     doc.size = 0;
-    const queries = !panel.ignore_global_filter ? req.payload.query : [];
-    const filters = !panel.ignore_global_filter ? req.payload.filters : [];
+    const ignoreGlobalFilter = panel.ignore_global_filter || series.ignore_global_filter;
```

Comment:
@stratoula @simianhacker
I'm not sure that It's too clear for me from the design perspective. Not sure that we can merge it cause it can confuse users.

Let's see my case: 
1) set some global filter and set `panel.ignore_global_filter` to true
2) create `2 aggs` and set different values  for `series.ignore_global_filter` (`true / false`)

I expect to see 2 different charts in `TSVB`: the first one should use global filter, the last one - no. 

I have 2 visions how to fix that: 
1) We can disable on UI `series.ignore_global_filter `if `panel.gnore_global_filter `is true
2) the default value for `series.ignore_global_filter` should be equal to `panel.gnore_global_filter `

Reply:
oh, you are right. It can be confusing. I like both proposals. Maybe the first one with an info tooltip will be the best from ux perspective. @simianhacker can you take care of this? Thank you!

Reply:
So what you're saying is that `series.ignore_global_filter` should override `panel.ignore_global_filter`? If `series.ignore_panel_filter` is set to false then it should ignore the panel setting?

Reply:
@simianhacker not exactly, I suggest to disable `series.ignore_global_filter` control if `panel.ignore_global_filter` is `true` with the corresponding tooltip. It will be super clear for users and avoid a lot of questions in future. 

Your current approach works differently than for example the `index pattern` field. In this case, if we define an `index pattern` for the series, it overrides the `index pattern` for the panel. `ignore_global_filter` works the other way around. This confuses me

Reply:
I can disable the series option when `panel.ignore_glboal_filter` is `true`, that's not a problem (I'm working on that now).

From a backend perspective, if the user sets `panel.ignore_global_filter: true` and somehow sends `series.ignore_global_filter: false` (maybe they are using the API directly), what should it do?  Add global filter or ignore it?

Reply:
I think yes, thank you

Reply:
"Yes, add global filter" OR "Yes, ignore it"

Reply:
I think we can ignore it. Do you agree with me?

Reply:
That's how it works now... if `panel.ignore_global_filters` is `true` OR if `series.ignore_global_filters` is `true` then it will ignore the global filters.

Reply:
yes, but `series.ignore_global_filters` should be disabled